runtimes/php/tutorial-symfony.md

```diff
@@ -109,3 +109,8 @@ if ($trustedProxies = $request->server->get('CC_REVERSE_PROXY_IPS')) {
 ```
 
 For more information on configuring symfony behind a reverse proxy, you can read the [official documentation](https://symfony.com/doc/current/deployment/proxies.html).
+
+### If you encounter 404 error after deployment
+
+If everything looks fine and you still get 404 errors, remember that CleverCloud works with an Apache server, so you'll need an htaccess in the  `/public` directory.
+Symfony got your back on this: just run `composer require symfony/apache-pack`. See https://symfony.com/doc/current/setup/web_server_configuration.html for more information.
```

Comment:
I think you need to do a markdown link here. There is an example a few lines above your changes: `[official documentation](https://symfony.com/doc/current/deployment/proxies.html).`